packages/integration.rb

```diff
@@ -27,12 +27,12 @@ def self.install
     system "gem install em-websocket"
     FileUtils.safe_unlink "#{CREW_PREFIX}/bin/mkdir"
     FileUtils.safe_unlink "#{CREW_PREFIX}/bin/install"
```

Comment:
I don't think this is a good idea.  We shouldn't be removing files from other packages.

Reply:
Removed